project.clj

```diff
@@ -33,14 +33,17 @@
                  [org.clojure/tools.cli "0.3.5"]
                  [environ "1.1.0"]
                  [aero "1.0.0"]
-                 [kixi/kixi.comms "0.1.21"]
+                 [kixi/kixi.comms "0.1.22"]
                  [org.clojure/tools.analyzer "0.6.9"]
                  ;; not really dependency, dep collision https://groups.google.com/forum/#!topic/clojure/D_s9Drua6D4
                  ]
   :plugins [[lein-ring "0.9.7"]]
   :pedantic? true
   :repl-options {:init-ns user}
-  :jvm-opts ["-XX:+HeapDumpOnOutOfMemoryError"]
+  :jvm-opts ["-XX:+HeapDumpOnOutOfMemoryError"
+             "-XX:-PrintGC"
```

Comment:
-Xloggc might be better, the kafka processes have good settings for the log rotation as well.